test/unit/data-feeds/DataFeedsUnit.t.sol

```diff
@@ -0,0 +1,228 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.19;
+
+import {Test, console2} from "forge-std/Test.sol";
+import {MockV3Aggregator} from "@chainlink/local/src/data-feeds/MockV3Aggregator.sol";
+import {MockOffchainAggregator} from "@chainlink/local/src/data-feeds/MockOffchainAggregator.sol";
+import {AggregatorV2V3Interface} from "@chainlink/local/src/data-feeds/interfaces/AggregatorV2V3Interface.sol";
+
+contract BasicDataConsumerV3Test is Test {
+    MockV3Aggregator public mockAggregator;
+    MockOffchainAggregator public mockOffchainAggregator;
+
+    uint8 public decimals;
+    int256 public initialAnswer;
+    uint256 public deploymentTimestamp;
+    uint80 public initialRoundId;
+
+    function setUp() public {
+        decimals = 8;
+        initialAnswer = 100000000000;
+        deploymentTimestamp = block.timestamp;
+        initialRoundId = 1;
```

Comment:
My understanding of the mock off chain aggregator is that the round id is intially at zero value, and gets incremented in updateAnswer() right?  i cant see that call happening  at construction time so how is the round id in the aggregator == 1 to match this initial setup value? when did it get updated in the aggregator?

so in line34 how is getAnswer returning a valid mapping value? when did the mapping get a value for roundId == 1 inserted into it?

Reply:
Ah got it.  In that case is there merit in removing line 22 and just assigning to 1 in line 16? optional/nit.